symbolic-symcache/src/writer.rs

```diff
@@ -18,22 +18,23 @@ fn is_empty_function(function: &Function<'_>) -> bool {
 }
 
 /// Performs a check whether this line has already been written in the scope of this function.
+/// NOTE: side effect?
```

Comment:
sure, you could rename this into `insert_line()` or something and document what it returns.

Reply:
At that point, couldn't it just be inlined?

Reply:
sure, why not